### ISSUE TITLE
Create wire draft of UI

### ISSUE THREAD

**Issue by jandegen** [amosproj/amos2023ss04-kubernetes-inventory-taker#76]
## User story
1. As a developer
2. I want / need a wire draft for the 
3. So that I know where to place frontend components

## Acceptance criteria
- [x] Wire draft contains a navigation concept
- [x] Wire draft contains detail page concept
- [x] Wire draft contains list view concept
- [x] Wire draft contains a landing page
- [x] Wire draft considers UX
- [x] Team agrees on design

## Linked Issues
- **Blocked by:**
- **Related to:**
- **Implements**

## Additional information
- PowerPoint draft is also fine

## Definition of done (DoD)
* Colours match with [color scheme](https://amos-p4.atlassian.net/l/cp/g5VzjL88)

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
